sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -290,6 +302,8 @@ internal BlobBaseClient(Uri blobUri, HttpPipelinePolicy authentication, BlobClie
             _version = options.Version;
             _clientDiagnostics = new ClientDiagnostics(options);
             _customerProvidedKey = options.CustomerProvidedKey;
+            _encryptionScope = options.EncryptionScope;
+            BlobErrors.VerifyCpkAndEncryptionScopeNotBothSet(_customerProvidedKey, _encryptionScope);
             BlobErrors.VerifyHttpsCustomerProvidedKey(_uri, _customerProvidedKey);
```

Comment:
Nit - the other .ctor verifies HTTPS before checking both CPK/EncryptionScope.  It'd be nice to throw errors in the same order.

Reply:
(Can defer)

Reply:
fixed.